index-v4.yaml

```
@@ -81,6 +81,12 @@ distributions:
     distribution_status: end-of-life
     distribution_type: ros1
     python_version: 2
+  jazzy:
+    distribution: [jazzy/distribution.yaml]
+    distribution_cache: http://repo.ros2.org/rosdistro_cache/jazzy-cache.yaml.gz
+    distribution_status: active
```

We should set this to pre-release for now:

```suggestion
    distribution_status: pre-release
```

(see also #36923 , where we did this for Iron)